crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,29 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_deadguardian
+TAGS:   no_monster_gen transparent
+MONS:   plant
+SUBST: z = t11
+KITEM: E = never_decay human skeleton / gnoll skeleton / elf skeleton / dwarf skeleton
```

Comment:
`never_decay` should be used very sparingly and with some kind of cue to the player that the corpse is special. I've suggested one take on this but you might have different ideas